services/avsregistry/avsregistry_chaincaller.go

```diff
@@ -76,11 +76,16 @@ func (ar *AvsRegistryServiceChainCaller) GetOperatorsAvsStateAtBlock(
 	}
 	numquorums := len(quorumNumbers)
 	if len(operatorsStakesInQuorums) != numquorums {
-		ar.logger.Fatal(
+		ar.logger.Error(
```

Comment:
I would add the values of `operatorsStakesInQuorums` and `numquorums` in this log for debugging

Reply:
Done